gremlin-language/src/main/antlr4/Gremlin.g4

```diff
@@ -1662,6 +1668,12 @@ infLiteral
     | SignedInfLiteral
     ;
 
+uuidLiteral
+    : K_UUID LPAREN RPAREN
+    | K_UUID LPAREN stringArgument RPAREN
```

Comment:
I would recommend using `stringLiteral` here instead of `stringArgument`. I don't see a need to allow users to pass a variable for the string inside the UUID.